CHANGELOG.md

```diff
@@ -1,11 +1,5 @@
 # Changelog
 
-## 1.0.1
-
-### Removes
-
-* Remove logs.
-
```

Comment:
That version has been unpublished right after being published. Really no risk that people could have been using it.
Tags also have been deleted.